packer/scripts/ubuntu/kernel-upgrade.sh

```diff
@@ -50,12 +50,12 @@ then
   sleep 2s
 
   echo "Installing the packages..."
-  dpkg -i "$DEB"
+  sudo dpkg -i $DEB
```

Comment:
add this DEBIAN_FRONTEND=noninteractive, so it would be sudo DEBIAN_FRONTEND=noninteractive dpkg -i $DEB